nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/java/org/nuxeo/ecm/platform/audit/service/LogEntryProvider.java

```diff
@@ -89,7 +89,11 @@ protected void doPersist(LogEntry entry) {
         Object entry = entries.get(0);
         if (entry instanceof LogEntry) {
             for (Object logEntry : entries) {
-                doPublish((LogEntry) logEntry);
+            	try {
+                    doPublish((LogEntry)logEntry);
+                } catch (Exception e) {
+                    log.error("unable to read log entry");
```

Comment:
Please don't catch `Exception` but `RuntimeException` or even a more specific one if there's a relevant one you know may happen. Also pass the exception `e` as second argument to `log.error` to get a stack trace.